stellar/docs/link-token.md

```diff
@@ -0,0 +1,230 @@
+```
+
+### Step 4: Link Token
+
+Link the token to the destination chain:
+
+```bash
+ts-node stellar/its.js link-token <salt> <destinationChain> <destinationTokenAddress> <tokenManagerType> \
+   --gas-amount <amount> \
+   --operator <operator>
+```
+
+## Examples
+
+### Example 1: Link USDC Tokens with LOCK_UNLOCK
+
+Link USDC tokens with different decimals (7 decimals on Stellar, 18 decimals on EVM):
+
+```bash
+# USDC already exists on both chains
+
+# Register USDC metadata on Stellar (7 decimals)
+ts-node stellar/its.js register-token-metadata CB64D3G...USDC --gas-amount 10000000
+
+# Register USDC metadata on EVM (18 decimals)
+ts-node evm/its.js --action registerTokenMetadata --tokenAddress 0xa0b86a33...USDC -n evm_chain
+
+# Register custom token on Stellar
+ts-node stellar/its.js register-custom-token 0x1234 CB64D3G...USDC LOCK_UNLOCK
+
+# Link token to EVM
+ts-node stellar/its.js link-token 0x1234 evm_chain 0xa0b86a33...USDC LOCK_UNLOCK --gas-amount 10000000
+
+# Interchain Token Transfer
+ts-node stellar/its.js interchain-transfer <tokenId> evm_chain <destinationAddress> <amount> --gas-amount 10000000
+```
+
+### Example 2: Link Custom Token with MINT_BURN
+
+```bash
+# Register custom token metadata on both chains
+ts-node stellar/its.js register-token-metadata <stellarTokenAddress> --gas-amount 10000000
+ts-node evm/its.js --action registerTokenMetadata --tokenAddress <ethereumTokenAddress> -n evm_chain
+
+# Register and link using MINT_BURN type
+ts-node stellar/its.js register-custom-token <salt> <stellarTokenAddress> MINT_BURN
+ts-node stellar/its.js link-token <salt> evm_chain <ethereumTokenAddress> MINT_BURN --gas-amount 10000000 --operator <operatorAddress>
+
+# Get token Manager on EVM
+ts-node evm/its.js --action tokenManagerAddress --tokenId <tokenId> -n evm_chain
+
+# Transfer mintership on EVM
+ts-node evm/its.js --action transferMintership --tokenAddress <tokenAddress> --minter <tokenManager> -n evm_chain
+
+# Interchain Token Transfer
+ts-node stellar/its.js interchain-transfer <tokenId> evm_chain <destinationAddress> <amount> --gas-amount 10000000
+```
+
+## Troubleshooting & Error Handling
+
+**Invalid Token Manager Type:**
+
+```
+Error: Invalid token manager type: INVALID_TYPE. Valid types: LOCK_UNLOCK, MINT_BURN
+```
+
+Solution: Use a valid token manager type from the supported list.
+
+**Token Not Registered:**
+
+```
+Error: Token metadata not found in ITS Hub
+```
+
+Solution: Register token metadata on both chains before linking.
+
+**Insufficient Gas:**
+
+```
+Error: Insufficient gas for cross-chain operation
+```
+
+Solution: Increase the gas amount using `--gas-amount` option.
+
+**Token Already Linked:**
+
+```
+Error: Token already linked with this salt
+```
+
+Solution: Use a different salt value for the linking operation.
+
+## Best Practices & Security
+
+1. **Salt Management**: Use unique, cryptographically secure salts for each token linking operation
```

Comment:
incorrect, salt can be any string, it just needs to be unique per token id being linked